abis/ERC20.json

```diff
@@ -0,0 +1,267 @@
+{
+  "abi": [
```

Comment:
@tnrdd Where was this ABI copied from?

Reply:
It's from the [app]( https://github.com/nation3/app/blob/main/ui/abis/ERC20.json) repo.